doorstop/server/main.py

```diff
@@ -23,6 +24,13 @@
 numbers = defaultdict(int)  # cache of next document numbers
 
 
+class HelpFormatter(argparse.ArgumentDefaultsHelpFormatter):
```

Comment:
Is there any reason not to use `ArgumentDefaultsHelpFormatter` as the base for `common.HelpFormatter`?